File: playwright-csharp/Common/TestAccount.cs
namespace PlaywrightStClimanuvem.Common;

/// <summary>
/// Account row used by parameterized system tests. Mirrors selenium-java's
/// <c>TestAccount</c>.
/// </summary>
public sealed record TestAccount(string Role, string Email, string Password, bool Verified, string Description)
{
    public override string ToString() => $"{Role}:{Email}";
}

File: puppeteer-python/climanuvem/common/test_account.py
# -*- coding: utf-8 -*-
"""Account row used by the system tests. Mirrors selenium-java's
``TestAccount``."""
from dataclasses import dataclass


@dataclass(frozen=True)
class TestAccount:
    role: str
    email: str
    password: str
    verified: bool
    description: str

    def __str__(self) -> str:
        return f"{self.role}:{self.email}"
